frontend/CHANGELOG.md

```diff
@@ -1,7 +1,10 @@
 # CHANGELOG
 
 ## vNext
```

Comment:
Please, move the content to this [CHANGELOG](https://github.com/Cerebellum-Network/cerestats/blob/dev-cere/CHANGELOG.md) and add the [FE] prefix before each item.

Reply:
fixed